grid-demo/src/app/angular/column-ordering/custom-clr-column-ordering-column.directive.ts

```diff
@@ -49,6 +49,19 @@ export class CustomClrColumnOrderingColumnDirective {
     }
   }
 
+  ngAfterViewInit(): void {
+    const separator = this.elementRef.nativeElement.querySelector('clr-dg-column-separator');
+    console.log(this.elementRef.nativeElement, 'sep');
+    if (separator) {
+      separator.addEventListener('mouseenter', () => {
+        this.cdkDrag.disabled = true;
+      });
+      separator.addEventListener('mouseleave', () => {
+        this.cdkDrag.disabled = false;
+      });
```

Comment:
We should remove these listeners in `ngOnDestroy`. The easiest way I know would be use `fromEvent` and unsubscribe from the observable like normal.